src/main/java/uk/gov/hmcts/reform/em/stitching/service/impl/DmStoreUriFormatter.java

```diff
@@ -17,7 +15,6 @@ public DmStoreUriFormatter(@Value("${dm-store-app.base-url}") String dmStoreAppB
     public String formatDmStoreUri(String s) {
         if (s.contains("/documents/")) {
             s = s.substring(s.indexOf("/documents/"));
-            s = ensureStringEndsWithSuffix(s, "/binary");
```

Comment:
how come this isn't required now?

Reply:
dm-store response from /documents/{UUID} is hateos json with a property called _links, which is a map of all associated resource URIs - one of them being "binary" - a URI to a binary - do we don't have to guess it

Reply:
https://restfulapi.net/hateoas/